client/src/redux/show-cert-saga.js

```diff
@@ -13,7 +12,8 @@ function* getShowCertSaga({ payload: { username, certSlug } }) {
       for (let i = 0; i < messages.length; i++) {
         yield put(createFlashMessage(messages[i]));
       }
-      yield call(navigate, '/');
+      // eslint-disable-next-line no-restricted-globals
+      history.go(-1);
```

Comment:
```suggestion
      yield call(navigate, '/');
```

Put this back for now so we can unblock this and move forward.